lib/splitio-api-binding.js

```diff
@@ -1,17 +1,51 @@
 'use strict'
 
+const https = require('https')
+
+const rp = require('request-promise')
```

Comment:
`request` is in maintenance mode: https://github.com/request/request/issues/3142

We should consider an alternative.  [axios](https://github.com/axios/axios) is a good candidate.